### PR TITLE
Use data containers instead of host mounts

### PR DESCRIPTION
**Pull request by bmoo**
Hi there,

This is the setup that works for me on my machine (by mounting to containers instead of the host). Feel free to incorporate any part of this that works for you. Cheers!

Brad

### CODE REVIEW

**Comment by pierreozoux** [comment]
Thanks, I don't have much time this week, I'll try next week to see your bug.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
   image: mysql
-  volumes:
-    - ./mysql/runtime:/var/lib/mysql
+  volumes_from:
+    - dbdata
   env_file:
     - ./.env
+
+dbdata:
+  image: mysql
```
Comment (snc) [inline docker-compose.yml]: This should cause problems when not overriding the entrypoint.

**Comment by bmoo** [comment]
It appears that my problems were do to volume mounts from the host not working correctly. Please reject this request and defer to the Docker folks for people having trouble with volume mounts.

**Comment by pierreozoux** [comment]
@bmoo it is what I thought, but didn't have time to investigate.
Thanks to let us know, I'll close this for now.

**Pull request closed without merging**